intyg-proxy-service/devops/dev/config/logback-spring.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<configuration>
+  <property name="APP_NAME" value="${APP_NAME:-intyg-proxy-}"/>
```

Comment:
Remove APP_NAME as it is not used for intyg-proxy-service.

Reply:
Removed